webui/react/src/e2e/models/components/ColumnPickerMenu.ts

```diff
@@ -1,6 +1,11 @@
-import { BaseComponent, CanBeParent, NamedComponent } from 'e2e/models/common/base/BaseComponent';
-import { BaseList, BaseRow } from 'e2e/models/common/base/BaseList';
-import { BasePage } from 'e2e/models/common/base/BasePage';
+import {
+  BaseComponent,
+  ComponentContainer,
+  NamedComponent,
+} from 'playwright-page-model-base/BaseComponent';
+import { BaseList, BaseRow } from 'playwright-page-model-base/BaseList';
+import { BasePage } from 'playwright-page-model-base/BasePage';
```

Comment:
Should this be shared base page or e2e/models/common/base/BasePage? I think I have a good guess about why, but kinda just highlighting that it's a bit hard to tell if this is the right base page with the current scheme.

Reply:
```
  columnPickerMenu = new ColumnPickerMenu({ parent: this, root: this.root });
```
when we provide root for the column picker, `this.root` is a base page. the constructor for dropdown needs some kind of base page, and we have access to base page through root. we could pass a determined page through by casting our root or replacing the function, but that's more work for no reason. we truly only need a base page here since we're not using `nav`